java/connecting-to-bigtable/src/test/java/com/example/cloud/bigtable/connecting/BigtableHelperTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example.cloud.bigtable.connecting;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class BigtableHelperTest {
+
+  // provide your project id as an env var
+  private final String projectId = System.getenv("GCLOUD_PROJECT");
```

Comment:
see earlier comment reg removing this and using system property.

Reply:
Done